corehq/apps/locations/fixtures.py

```diff
@@ -142,7 +163,7 @@ def _get_metadata_node(location):
 
 
 def _location_to_fixture(location_db, location, type_lookup_function):
-    root = Element(type_lookup_function(location.location_type), {'id': location._id})
+    root = Element(type_lookup_function(location.location_type.name), {'id': location.location_id})
```

Comment:
Good idea. I actually just passed the type along (instead of the lookup function) to prevent duplicate queries.